packages/db/prisma/schema.prisma

```diff
@@ -49,8 +49,19 @@ model User {
   image         String?
   accounts      Account[]
   sessions      Session[]
+  bookmark      Bookmark[]
 }
 
+model Bookmark {
+  id      String @id @default(uuid())
+  user    String
+  userID  User @relation(fields: [user],references: [id])
```

Comment:
could add onDelete: Cascade on line 58 and 60, to delete the bookmark table if a particular user deletes his/her account.

Reply:
Thanks for the suggestion , totally missed out. I've made the changes now. Let me know if you have any other suggestion :)